### PR TITLE
Add install task

### PR DESCRIPTION
**Pull request by sabljak**
New meilisearch:install task creates an initializer file

# Pull Request

## What does this PR do?
- adds a meilisearch:install task to create the meilisearch.rb initializer with template configuration
- adds changes to README.md file to include environment variables

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: lib/meilisearch/rails/templates/initializer.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+MeiliSearch::Rails.configuration = {
+  meilisearch_url: ENV.fetch("MEILISEARCH_HOST", "YourMeilisearchUrl"), # example: http://localhost:7700
```
Comment (brunoocasali) [inline lib/meilisearch/rails/templates/initializer.rb:2]:
```suggestion
  meilisearch_url: ENV.fetch("MEILISEARCH_HOST", "http://localhost:7700"),
```

**Review by brunoocasali — CHANGES_REQUESTED** [review]
I liked it a lot, very nice to see it! I just had one comment:

I think it is better to provide a realistic sample like the `http://localhost:7700`

File: lib/meilisearch/rails/templates/initializer.rb (hunk 1 above)
Comment (sabljak) [reply lib/meilisearch/rails/templates/initializer.rb:2]: Thanks for the feedback!

Nice catch! I'll add it right away.

**Comment by sabljak** [comment]
@brunoocasali fixed!

Should we add the same to README.md or leave it as is?

**Comment by brunoocasali** [comment]
> Should we add the same to README.md or leave it as is?

I think it is good to have it in the README!

**Comment by brunoocasali** [comment]
@sabljak I'll wait for the change before merging, thanks a lot!

**Comment by sabljak** [comment]
@brunoocasali Done! Ready for merge 😁

**Review by brunoocasali — APPROVED** [review]
bors merge

**Pull request merged**